cfgen/cfgen

```diff
@@ -219,42 +237,82 @@ fdmi_filters:  # This section is optional.  If it is missing, no fdmi filters
         sys.exit()
 
 
+def create_logger_directory(log_dir):
+    """Create log directory if not exists."""
+    if not os.path.isdir(log_dir):
+        try:
+            os.makedirs(log_dir)
+        except Exception as e:
+            logging.exception(f"{log_dir} Could not be created: {e}")
+            raise
+
+
+def setup_logging(opts: Any) -> None:
+    log_dir = opts.log_dir
+    max_size = 1024 * 1024
+    handlers: List[logging.Handler] = []
+    if log_dir:
+        log_file = f'{log_dir}/cfgen.log'
```

Comment:
This will go in a separate log file and not in the `hare_deployment/setup.log` e.g. `/etc/cortx/var/log/cortx/hare/log/92abaf0e61574526b3f627ad5726e7a6/hare_deployment/setup.log`. We need to add this to the support bundle as well. I wonder if we must use a single log file instead of multiple.

Reply:
I used to think that support bundle takes the whole log directory (the one which is provided to `configure` script via `--log-dir` parameter). Is that incorrect?

Reply:
I mean collecting  logs is okay but having a single log file for deployment is more user friendly than having to multiple ones.

Reply:
Done. When `configure` script invokes `cfgen`, it provides the `setup.log` filename that is the same one to `hare_setup`.